pkg/util/autopilot.go

```diff
@@ -49,6 +49,11 @@ var AutopilotManagedKinds = []schema.GroupVersionKind{
 	admissionregistrationv1.SchemeGroupVersion.WithKind("MutatingWebhookConfigurationList"),
 }
 
+var autopilotWebhooks = []string{
+	"policycontrollerv2.config.common-webhooks.networking.gke.io",
+	"gkepolicy.config.common-webhooks.networking.gke.io",
```

Comment:
Please add some comments here: which one is new and when it is introduced, which one is legacy.

Reply:
Added a comment about which one is new and which is legacy. I'm still confirming with the autopilot team to see which version was this introduced (waiting for the response). Will add the start version when I get an answer.